src/react-components/code-sample.tsx

```diff
@@ -33,59 +41,79 @@ export type CodeSamplesViewerProps = {
    * the component. Receives data about the selected code sample. The library
    * comes pre-packaged with some sensible options.
    *
-   * @see CodeSampleMethodTitle
+   * @see CodeSampleTitle
    * @see CodeSampleFilenameTitle
    * @default CodeSampleMethodTitle
    */
-  title?: CodeSampleTitleComponent | React.ReactNode | string;
-  /** The operation to get a code sample for. Can be queried by either
-   * operationId or method+path.
+  title?: CodeSampleTitleComponent | React.ReactNode | string | "none";
+  /** The operations to get code samples for. If only one is provided, no selector will be shown.
+   * Can be queried by either operationId or method+path.
    */
-  operation: MethodPaths | OperationId;
+  operations?: MethodPaths[] | OperationId[];
```

Comment:
Do we want to allow users to mix operation id's and methodpaths? If so, this could probably be changed:
```suggestion
  operations?: (MethodPaths | OperationId)[];
```

Edit: looks like the logic below wants the user to pick one and stick with it 👍🏻

Reply:
I believe that's required by the API

Reply:
it's not required - i made it so it would work with a mix. i'm cool with us assuming they'll pick one and stick w/ it though.